README.md

```diff
@@ -69,6 +69,42 @@ modify the contents of the `./templates` directory directly, as they will
 be overwritten during the next application update. Files under
 `./state/webtop.properties.d/` are retained and are also added to backup.
 
+## Phonebook feature
+
+The phonebook synchronization system allows you to manage contacts between
+**WebTop** and a centralized **Phonebook**. This feature is supported by
+scripts and timer-based systemd services for automation.
+
+### Synchronization Scripts Overview
+
+- **`webtop2phonebook.php`:**
+   - Extracts contacts from WebTop.
+   - Processes the contacts and removes pre-existing WebTop entries in the
+     phonebook.
+   - Inserts the cleaned and updated contacts into the phonebook database.
+
+- **`pbook2webtop.php`:**
+   - Extracts contacts from the phonebook database.
+   - Cleans up the WebTop contacts folder and replaces its contents with
+     updated contacts.
+
+Both scripts use environment variables (`PHONEBOOK_DB_HOST`,
+`PHONEBOOK_DB_PASSWORD`) for connecting to the databases. The sync process to
+run daily at 23:00.
+
+### Manual Configuration Steps
+
+Follow these steps to manually configure and use the phonebook synchronization:
+1. Set the environment variables for the phonebook database connection.
+  - **`PHONEBOOK_DB_HOST`**: The hostname or IP address of the phonebook
+    database, formatted as `host:port`.
+  - **`PHONEBOOK_DB_PASSWORD`**: The password for the phonebook database user.
```

Comment:
Add an example here.
Document also that this configuration is not preserved when the application is restored or moved/cloned.

Possible improvements: add such variables into a new env file, load the envfile from the phonebook unit and add it also to backup.